doc/plotting.rst

```diff
@@ -375,9 +375,9 @@ Faceted plotting supports other arguments common to xray 2d plots.
     hasoutliers[0, 0, 0] = -100
     hasoutliers[-1, -1, -1] = 400
 
-    @savefig plot_facet_robust.png height=12in 
-    g = hasoutliers.plot.contourf('lon', 'lat', col='time', col_wrap=3,
-                                  robust=True, cmap='viridis')
+    @savefig plot_facet_robust.png height=12in
+    g = hasoutliers.plot.imshow('lon', 'lat', col='time', col_wrap=3,
+                                robust=True, cmap='viridis')
 
```

Comment:
`pcolormesh` is upside down, because that's what's required to be consistent with matplotlib defaults. For xray plotting it might make sense that all of them have the same orientation, regardless of plot type.

![image](https://cloud.githubusercontent.com/assets/5356122/10237493/1ec3dfaa-6867-11e5-8252-745eb58edecd.png)